manifests/4.5/sriov-network-operator-sriovnetworknodestate.crd.yaml

```diff
@@ -92,14 +86,18 @@ spec:
                         mac:
                           type: string
                         mtu:
-                          format: int64
```

Comment:
why this is removed?

Reply:
This CRD file is generated by the latest version operator-sdk. I guess `format` is not necessary.